teku/src/test/resources/P2POptions_config.yaml

```diff
@@ -7,6 +7,7 @@ p2p-advertised-port: 9999
 p2p-private-key-file: "/the/file"
 p2p-advertised-ip: "127.200.0.1"
 p2p-static-peers: "127.1.0.1,127.1.1.1"
+p2p-static-peers-url: "file:///path/to/peers.txt"
```

Comment:
Just to confirm, using `ResourceLoader.urlOrFile()` means we don't need the `file://` when dealing with filesystem, right?

Reply:
i've also removed this i thought, checking also